scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -602,11 +611,12 @@ object Execution {
         futCancel1 <- Trampoline.call(one.runStats(conf, mode, cache))
         futCancel2 <- Trampoline.call(two.runStats(conf, mode, cache))
       } yield {
-        val (f1, cancelHandler1) = futCancel1
-        val (f2, cancelHandler2) = futCancel2
+        val CFuture(f1, cancelHandler1) = futCancel1
+        val CFuture(f2, cancelHandler2) = futCancel2
```

Comment:
can we move this to `CFuture.zip`? Seems like we can simplify the code if we push it there.

Reply:
👍